test/plugin/test_out_typecast.rb

```diff
@@ -71,6 +72,20 @@ def test_typecast_float
     assert_equal(v, record['f'])
   end
 
+  def test_typecast_json
+    d = create_driver(DEFAULT_CONFIG + %[
+      tag test.tag
+      item_types j:json
+    ])
+    v = {"msg" => "ok"}
+    time = Time.parse('2015-01-19 08:35:15 UTC').to_i
+    d.run do
+      d.emit({'j' => v }, time)
```

Comment:
Space inside { missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.